core/modules/views/config/views.view.promoted.json

```diff
@@ -1,12 +1,13 @@
 {
-    "_config_name": "views.view.frontpage",
-    "name": "frontpage",
-    "description": "Emulates the default Backdrop front page; you may set the default home page path to this view to make it your front page.",
+    "_config_name": "views.view.promoted",
+    "name": "promoted",
+    "description": "Provides a list of promoted content for the default Backdrop front page.",
```

Comment:
I think avoiding the word "Backdrop" in the UI is good whenever possible. So perhaps something like "Provides a list of promoted content. Used by default on the front page."